### DISCUSSION TITLE
Subset a font for including in a pdf

### DISCUSSION THREAD

**Discussion by HJ1967** [harfbuzz/harfbuzz#3193] [Q&A] [answered]
Is it possible to get the glyphs for certain unicode value at a fixed position.

For example I want the 'A' as glyph id 1 in the file and the 'm' as glyhid 2.

**Comment by khaledhosny** [comment, marked as answer]
I don’t think this is currently supported, see https://github.com/harfbuzz/harfbuzz/issues/3140.

**Reply by HJ1967** [reply]
do you know if this enhancement will be implement in the next version?

**Reply by khaledhosny** [reply]
No idea.

**Reply by behdad** [reply]
Not next version, but eventually.

**Reply by garretrieger** [reply]
In the mean time you can use the retain gids option. Which will cause glyph ids to not change between the original font and the subset. So if A is 1 in the original font in the subset it will also be 1.

**Reply by HJ1967** [reply]
Well I want to remap the ids because input is limited to 255 options and there are more then 255 glyphs.
